scss/_nav.scss

```diff
@@ -74,12 +74,13 @@
 .nav-pills {
   .nav-link {
     @include border-radius($nav-pills-border-radius);
+  }
 
-    &.active,
-    .show & {
-      color: $nav-pills-active-link-color;
-      background-color: $nav-pills-active-link-bg;
-    }
+  .nav-link[aria-expanded="true"],
+  .nav-link.active,
+  .nav-item.show .nav-link {
```

Comment:
Consider removing the `.nav-item` from `.nav-item.show .nav-link` so it's a slightly less specific selector?